css/style.less

```
@@ -0,0 +1,24 @@
+#game-container {
+    padding: 20px;
+    font-family: sans-serif;
+}
+
+#board {
+    display: grid;
+    // Tạo lưới 3 hàng, 5 cột
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)